stdlib/2and3/csv.pyi

```diff
@@ -22,25 +22,25 @@ _Dialect = Union[str, Dialect]
 _DictRow = Dict[str, Any]
 
 class Dialect(object):
-    delimiter = ...  # type: str
-    quotechar = ...  # type: Optional[str]
-    escapechar = ...  # type: Optional[str]
+    delimiter = ...  # type: Text
```

Comment:
This kind of thing can also break working code if people rely on `Dialect.delimiter` actually being a `str` in Python 2. Maybe these attributes need to be Any or str, unfortunately.

Does your code work if you use `Text` just in argument types?